attest.js

```diff
@@ -258,18 +259,22 @@ function postAndWriteAttestation(device_address, attestor_address, attestation_p
 	});
 }
 
+function handleMessage(device_address, user_address) {
+	if (!ValidationUtils.isValidAddress(user_address.trim()))
+		sendMessageToDevice(device_address, 'To get attestation profile, let me know your Obyte address (use "Insert My Address" button)');
```

Comment:
hey @tarmo888 , didn't you forget a `return` here? Otherwise the code below executes even if the address is invalid.